### PR TITLE
WDP190705-11

### PR DESCRIPTION
**Pull request by morales1980**
Add RWD to header main menu

### CODE REVIEW

File: src/sass/components/_header.scss [hunk 1]
```diff
@@ -126,6 +126,108 @@ header {
       height: 84px;
```
Comment (EnDriash) [inline src/sass/components/_header.scss]: duplicated file!  _header.scss was moved to section directory.
git pull is required

File: src/sass/components/_header.scss [hunk 2]
```diff
@@ -126,6 +126,108 @@ header {
...
     }
```
Comment (EnDriash) [inline src/sass/components/_header.scss]: Don't mixed media-queries/RWD with plain scss.
You have got _media-queries file in src/sass directory, please move your code there.
Create header selector in intrested range and put it there.

File: src/scripts/app.js [hunk 3]
```diff
@@ -1 +1,22 @@
...
+{
+  // Header main menu open animation
+
+  const mobileMainMenuTrigger = document.querySelector('.mobile-menu-trigger-button');
+  const mobileMainMenu = document.querySelector('.menu-bar');
+  const mobileMainMenuActiveItem = document.querySelector(
+    '.menu > ul > li:first-child a'
+  );
+
+  const showMainMenu = () => {
```
Comment (EnDriash) [inline src/scripts/app.js]: showMainMenu is a reference to your functional expression.
create funcion or move your funcional expression to EventListener.

File: src/scripts/app.js [hunk 4]
```diff
@@ -1 +1,22 @@
 console.log('hello');
```
Comment (EnDriash) [inline src/scripts/app.js]: This code is only for menu in header section. The better way will be create individual file mobile header.js or sth like this.

File: src/scripts/app.js (hunk 3 above)
Comment (morales1980) [reply src/scripts/app.js]: showMainMenu is ES6 arrow function declaration.
Keepeing code readaibility in mind, imho, it is better to first declare function and then bind it to eventListenter.

File: src/scripts/app.js (hunk 4 above)
Comment (morales1980) [reply src/scripts/app.js]: done as suggested

File: src/sass/components/_header.scss (hunk 1 above)
Comment (morales1980) [reply src/sass/components/_header.scss]: resolved

File: src/sass/components/_header.scss (hunk 2 above)
Comment (morales1980) [reply src/sass/components/_header.scss]: resolved

File: src/scripts/app.js (hunk 3 above)
Comment (morales1980) [reply src/scripts/app.js]: resolved

Comment (EnDriash) [reply src/scripts/app.js]: you don't declared function, you declared reference to functional expression. it is a difference.

**Pull request merged**